src/libraries/Microsoft.PowerFx.Interpreter/IRuntimeContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    internal interface IRuntimeContext
```

Comment:
>IRuntimeContext [](http://example.com/codeflow?start=23&length=15)

Can we shrink this interface?

This is a lot of information to pass around everywhere. 

It's also useful to separate CancellationToken from the rest. 
CT is a "single use" object (once signalled, it's done). Whereas everything else can potentially be shared across many evals.

Reply:
Done